source/_layouts/post.blade.php

```diff
@@ -32,13 +32,13 @@
                       <img src="https://www.gravatar.com/avatar/{{$page->gravatar}}?size=40"
                       alt="{{ $page->author }}" />
                       <span>
-                        By <a href="{{ $page->baseUrl }}team/{{ \Illuminate\Support\Str::slug($page->author) }}"> {{ $page->author }} </a>
+                        {{ $page->t("By") }} <a href="{{ $page->baseUrl }}team/{{ \Illuminate\Support\Str::slug($page->author) }}"> {{ $page->author }} </a>
```

Comment:
Will be best to translators if the string to translate contains a reference to the author name.
```suggestion
                        {{ $page->t('By {authorName}') }} <a href="{{ $page->baseUrl }}team/{{ \Illuminate\Support\Str::slug($page->author) }}"> {{ $page->author }} </a>
```

To do this or we will need to change the t function to implement the algorithm to do this replace or we will need to assign the translated var to a string and after this, make the replace. I think that would be best to put inside the t function.